doc/HttpLuaModule.wiki

```diff
@@ -3069,6 +3069,14 @@ Then <code>GET /orig</code> will give
 
 rather than the original <code>"hello"</code> value.
 
+Because HTTP request is created after SSL handshake, the <code>ngx.ctx</code> created
+in [[#ssl_certificate_by_lua|ssl_certificate_by_lua*]], [[#ssl_session_store_by_lua|ssl_session_store_by_lua*]] and [[#ssl_session_fetch_by_lua|ssl_session_fetch_by_lua*]]
+is not available in the following phases like [[#rewrite_by_lua|rewrite_by_lua*]].
+
+Since <code>dev</code>, the <code>ngx.ctx</code> created during a SSL handshake
+will be inherited by the requests which share the same TCP connection established by the handshake.
+Note that the inherited value from <code>ngx.ctx</code> is read-only.
```

Comment:
```suggestion
Note that overwrite values in <code>ngx.ctx</code> in the http request phases (like `rewrite_by_lua*`) will only take affect in the current http request.
```

Reply:
and add the following case will be much clearer.

for example:

```
ssl_certificate_by_lua_by_block {
    ngx.ctx.foo = 1  -- it will always be 1 in all of the http requests over this SSL connection.
}

location /foo-1 {
    rewrite_by_lua_block {
        ngx.ctx.foo = 2 -- only take affects in current http request
    }
}

location /foo-2 {
    rewrite_by_lua_block {
        ngx.say(ngx.ctx.foo) -- will alway be 1
    }
}
```

Reply:
@doujiang24 
Done. Test case is added in https://github.com/openresty/lua-resty-core/pull/243